ceph_iscsi_config/settings.py

```diff
@@ -51,7 +51,41 @@ class Settings(object):
         "max_data_area_mb": {
             "min": 1, "max": 2048},
         "osd_op_timeout": {
-            "min": 0, "max": 600}
+            "min": 0, "max": 600},
+
+        "emulate_3pc": {
+            "min": 0, "max": 1},
+        "emulate_caw": {
+            "min": 0, "max": 1},
+        "emulate_dpo": {
+            "min": 0, "max": 1},
+        "emulate_fua_read": {
+            "min": 0, "max": 1},
+        "emulate_fua_write": {
+            "min": 0, "max": 1},
+        "emulate_model_alias": {
+            "min": 0, "max": 1},
```

Comment:
```emulate_model_alias``` is not a boolean. It corresponds to the T10 INQUIRY MODEL string, which can be up to 16 bytes long.

Reply:
Thanks @ddiss, since `emulate_model_alias` is a `string`, can you please just confirm if documentation is still correct here:
https://github.com/SUSE/doc-ses/blob/master/xml/deployment_iscsi.xml#L614

Reply:
The documentation is indeed incorrect. I'll raise a ticket to track this.

Reply:
@ddiss I've done the following test and I was not able to set a value different then `0` or `1`:

```
/backstores/rbd/rbd.lun1> get attribute emulate_model_alias
emulate_model_alias=0 
/backstores/rbd/rbd.lun1> set attribute emulate_model_alias=1
Parameter emulate_model_alias is now '1'.
/backstores/rbd/rbd.lun1> get attribute emulate_model_alias
emulate_model_alias=1 
/backstores/rbd/rbd.lun1> set attribute emulate_model_alias="test"
Parameter emulate_model_alias is now '0'.
/backstores/rbd/rbd.lun1> get attribute emulate_model_alias
emulate_model_alias=0
```

Am I doing something wrong?